source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
```

Comment:
Thanks @jtappa! I wasn't trying to incite a whole rewrite at this very moment, but it is something to keep in mind. I've noticed this language in other docs as well, referring to hiera and facter and puppet packages when we don't provide individual packages for any of those project anymore. We make them all available as a part of the puppet-agent package. It's definitely a shift in how we and users are communicating, and everyone's still getting used to it :(